src/app/datagrid/selection-single/selection-single.html

```diff
@@ -102,7 +106,11 @@ <h3>Client side, trackBy item</h3>
   <clr-dg-column>Creation date</clr-dg-column>
   <clr-dg-column>Favorite color</clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of trackByIdUsers; trackBy: trackById" [clrDgItem]="user">
+  <clr-dg-row
+    *clrDgItems="let user of trackByIdUsers; trackBy: trackById"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.